realm/realm-library/src/main/cpp/.clang-format

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
Can we reference the one in cores repo instead? Then we don't have to maintain a copy here?